akka-http-core/src/main/scala/akka/http/javadsl/Http.scala

```diff
@@ -40,6 +42,7 @@ class Http(system: ExtendedActorSystem) extends akka.actor.Extension {
 
   import language.implicitConversions
   private implicit def completionStageCovariant[T, U >: T](in: CompletionStage[T]): CompletionStage[U] = in.asInstanceOf[CompletionStage[U]]
+  @silent("never used")
```

Comment:
Why is it needed?

Reply:
The implicit 'evidence' parameter `ev` is not actually used, so the compiler emits a warning for this line.

Of course introducing the silencer compiler plugin just for this one warning wouldn't make sense, but I think it's a nice tool to be able to suppress spurious warnings going forward in general.